commands/_helper.js

```diff
@@ -1,3 +1,19 @@
+const Discord = require("discord.js");
+
 exports.asyncHandler = (cb) => (argv) => {
   argv.handled = cb(argv);
 };
+
+// Discord message "multiple items found"
+exports.multipleResultsFoundMessage = (searchTerm, suggestions) =>
+  new Discord.RichEmbed()
+    .setTitle(`Multiple results for '${searchTerm}'`)
+    .addField('Did you mean:', `${suggestions}`)
+    .setThumbnail('https://questland-discord-bot.cfapps.io/sherlock.png');
+
+// Discord message "no item results found"
+exports.noResultFoundMessage = (searchTerm, searchCategory) =>
+  new Discord.RichEmbed()
+    .setTitle(`${searchCategory} '${searchTerm}' not found`)
+    .addField('Please check your input', '\u200b')
+    .setThumbnail('https://questland-discord-bot.cfapps.io/noidea.png');
```

Comment:
Where do you want this? In my usual coding, I would create a new folder "Helpers", and a new file "messageHelper.js" in a case like this.

Reply:
Yeah I am good with a helper directory here it seems like a solid choice to me

Reply:
You can leave the _helper file there since that is specific to how I have asynchronous commands managed